### ISSUE TITLE
There is no 'stack' command in docker --help but I am prompted for this command when deploying swarm

### ISSUE THREAD

**Issue by moocherabc** [docker/for-linux#278]
There is no 'stack' command in docker --help but I am prompted for this command when deploying swarm, docker stack --help shows this command... docker --help is missing this 'stack' command....

 docker -v 
Docker version 18.03.0-ce, build 0520e24

![image](https://user-images.githubusercontent.com/31909570/38554575-b10e0fba-3cf5-11e8-8b6e-ed3289684f9a.png)

![image](https://user-images.githubusercontent.com/31909570/38554543-8d3b93e6-3cf5-11e8-8405-2542c55b69d8.png)

Docker version 17.12.0-ce, build c97c6d6    Is there this order

**Comment by vdemeester** [comment]
cc @thaJeztah @silvin-lubecki

**Comment by thaJeztah** [comment]
Yup, I can reproduce. Not sure what's causing it; template is here; https://github.com/docker/cli/blob/a6d235430a0a69f6515606604d63252ab20510c1/cli/cobra.go#L137-L145

And using this function to add commands to the list; https://github.com/docker/cli/blob/a6d235430a0a69f6515606604d63252ab20510c1/cli/cobra.go#L92-L100

**Comment by silvin-lubecki** [comment]
I think the bug comes from the stack command which has 2 feature annotations "swarm" and "kubernetes" https://github.com/docker/cli/blob/master/cli/command/stack/cmd.go#L17 
And here https://github.com/docker/cli/blob/master/cmd/docker/docker.go#L239 we hide one or the other feature...
I'm fixing it!

**Comment by thaJeztah** [comment]
Ah! Yes I recall that one; thanks!
